tests/InputFiles/ScalarWave/SphericalWave3D.yaml

```diff
@@ -0,0 +1,57 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveRegularSphericalWave3D
+# Check: parse;execute
+
+AnalyticSolution:
+  RegularSphericalWave:
+    Profile:
+      Gaussian:
+        Amplitude: 1.0
+        Width: 1.0
+        Center: 0.0
+
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.001
+  InitialSlabSize: 0.01
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 3
+  StepController: BinaryFraction
+  StepChoosers:
+    - Constant: 0.05
+    - Increase:
+        Factor: 2
+    - Cfl:
+        SafetyFactor: 0.2
+
+DomainCreator:
+  Shell:
```

Comment:
Since the solution is regular at the origin you can make this a `Sphere`

Reply:
changed